app/controllers/main_controller.rb

```diff
@@ -0,0 +1,4 @@
+class MainController < ApplicationController
+  def index
+  end
```

Comment:
Use single line syntax for empty actions
```suggestion
  def index; end
```